packages/app-admin-ui/client/pages/Item/AddNewItem.js

```diff
@@ -7,10 +7,13 @@ import { useList } from '../../providers/List';
 
 const AddNewItem = () => {
   let {
-    list: { access },
+    list: {
+      access,
+      adminConfig: { singleton = false },
+    },
     openCreateItemModal,
   } = useList();
-  if (!access.create) return null;
+  if (!access.create || singleton) return null;
```

Comment:
If we force `{ create: false }` for the access on this list (see other comment), then we don't need to make this change as we'll get the behaviour we want for free.

Reply:
you are right.